[CS2103T-F11-3] SecureIT

hooncp opened the pull request:
@jityong
@niqiukun
@yhtingg
@eejian97

e0031374 commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,56 +301,479 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+---
+
+[discrete]
+=== UC21 - Encrypt a file
```
Could be missing extension, 1a where file to encrypt is missing

e0031374 commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,56 +301,479 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+* 2a. Details entered are invalid
++
+[none]
+** 2a1. FileSys shows an error message
```
Could be an unnecessary UI detail mentioned might want to try: "notify user" instead of "shows error message" which might be a bit specific

stevenwjy commented on docs/DeveloperGuide.adoc:
```diff
@@ -279,13 +279,19 @@ Refer to the guide <<DevOps#, here>>.
+*Value proposition*: Remember only one password, and save the hassle of remembering all other confidential documents (account details, credit card details, secret files, secret notes). Have a safe and secure way to store all confidential documents locally, without the use of the online/ third party / cloud-reliant vaults.
 
 [appendix]
 == User Stories
```
Our team has read through your user stories and we notice that some important ones might be missing.

Must have:
- List passwords
- Find passwords associated with some accounts
- List credit card info
- Add new file to encryption system

Nice to have:
- Two-factor authentication

stevenwjy commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,56 +301,479 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
```
Some of your user stories are in the wrong format, e.g. `I can be sure ...`.

e0031374 commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,56 +301,479 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+---
+[discrete]
+=== UC11 - Add a password
```
Might be missing a guarantee: "password is added"

stevenwjy commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,56 +301,479 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+Reliability
+
+* The app shall throw appropriate exceptions when any user input is invalid or any user action fails to execute completely.
```
Maybe it would be better if you could paraphrase this into `the system has to be robust to errors`, because an NFR should not be too detail.

stevenwjy commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,56 +301,479 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+Data Integrity
+
+* Should be able to check for the data integrity as to verify that no one has modified the files within secureIT in an unauthorised fashion.
```
This sounds too functional, maybe it would be better to rephrase the sentence into `this program should consider data integrity of files`.

shiweing commented on docs/DeveloperGuide.adoc:
```diff
@@ -98,9 +93,9 @@ image::LogicClassDiagram.png[]
 *API* :
```
The text in the diagrams seems too small, perhaps the arrows could be shorter so the diagram can be more zoomed in?
![image](https://user-images.githubusercontent.com/26473264/67543141-898a4600-f722-11e9-99fe-5707d1fc41e0.png)

shiweing commented on docs/DeveloperGuide.adoc:
```diff
@@ -109,7 +104,7 @@ Given below is the Sequence Diagram for interactions within the `Logic` componen
+NOTE: The lifeline for `DeleteNoteCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
 
 [[Design-Model]]
 === Model component
```
The text in the diagram seems to be too small as well and the diagram appears a bit cluttered. Perhaps the Model class diagram could be split into sub components for Card, Notes etc.
![image](https://user-images.githubusercontent.com/26473264/67543190-c9512d80-f722-11e9-8c18-55ccdda30904.png)

Nanosync replied:
The diagram shows a good overview of all the components, however it is a bit too small. Maybe it could be broken into smaller components.

shiweing commented on docs/DeveloperGuide.adoc:
```diff
@@ -154,89 +145,86 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
-[NOTE]
-If the `currentStatePointer` is at index 0, pointing to the initial address book state, then there are no previous address book states to restore. The `undo` command uses `Model#canUndoAddressBook()` to check if this is the case. If so, it will return an error to the user rather than attempting to perform the undo.
+image::AnalysePasswordSequenceDiagram.png[]
```
The activation bar for DictionaryAnalyser self-called method getAllMatches() seems to be missing a return arrow. Also, the DictionaryMatch return arrow would require the return arrow head to be pointing within the activation bar instead of at the end of the bar.
![image](https://user-images.githubusercontent.com/26473264/67543437-f225f280-f723-11e9-81b6-def2461ff3ac.png)

shiweing commented on docs/DeveloperGuide.adoc:
```diff
@@ -154,89 +145,86 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
-The undo/redo mechanism is facilitated by `VersionedAddressBook`.
-It extends `AddressBook` with an undo/redo history, stored internally as an `addressBookStateList` and `currentStatePointer`.
-Additionally, it implements the following operations:
+image::AnalyserClassDiagram.png[]
```
A figure caption seems to be missing for a few diagrams.

Nanosync replied:
The diagram can be resized.
It appears too verbose.
Each component could be described individually.
Missing figure caption.

Nanosync replied:
The `Match` direction seems incorrect, did you mean `Match` implements BaseMatch and DictionaryMatch extends BaseMatch?

Nanosync commented on docs/DeveloperGuide.adoc:
```diff
@@ -98,9 +93,9 @@ image::LogicClassDiagram.png[]
 *API* :
 link:{repoURL}/src/main/java/seedu/address/logic/Logic.java[`Logic.java`]
```
Diagram can be broken down into smaller parts

Nanosync commented on docs/DeveloperGuide.adoc:
```diff
@@ -14,7 +14,7 @@ ifdef::env-github[]
 :repoURL: https://github.com/se-edu/addressbook-level3/tree/master
 
-By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
+By: `SecureIT`      Since: `Jun 2016`      Licence: `MIT`
```
The date appears incorrect.

Nanosync commented on docs/DeveloperGuide.adoc:
```diff
@@ -109,7 +104,7 @@ Given below is the Sequence Diagram for interactions within the `Logic` componen
 .Interactions Inside the Logic Component for the `delete 1` Command
 image::DeleteSequenceDiagram.png[]
```
The diagram shows `AddressBookParser`. Also, since there are multiple directories, the context of `delete` is not explicitly mentioned in the preface. There is also no description to describe the diagram.

Nanosync commented on docs/DeveloperGuide.adoc:
```diff
@@ -154,89 +145,86 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
-* **Alternative 2:** Individual command knows how to undo/redo by itself.
-** Pros: Will use less memory (e.g. for `delete`, just save the person being deleted).
-** Cons: We must ensure that the implementation of each individual command are correct.
+image::InitPasswordSequenceDiagram.png[]
```
There are missing activation bars for Storage, Model, Logic and Ui after they are constructed.

Nanosync commented on docs/DeveloperGuide.adoc:
```diff
@@ -154,89 +145,86 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
-** Pros: Will use less memory (e.g. for `delete`, just save the person being deleted).
-** Cons: We must ensure that the implementation of each individual command are correct.
+image::InitPasswordSequenceDiagram.png[]
 
```
We think it would be helpful to have an activity diagram to summarise the feature.

Nanosync reviewed (commented):
Some changes might help to improve the document.

niqiukun committed:
Add static class labels to sequence diagrams

niqiukun committed:
Update file section of the user guide

hooncp committed:
Fix multiple fields and limit to tag

hooncp committed:
Merge pull request #93 from AY1920S1-CS2103T-F11-3/fix-password-branch

Fix password branch

niqiukun committed:
Fix file duplication when file is being opened in Windows before encryption

hooncp committed:
Update command format tool tip

niqiukun committed:
Merge pull request #95 from AY1920S1-CS2103T-F11-3/fix-windows-file

Fix file duplication when file is being opened before encryption

eejian97 committed:
added font

eejian97 committed:
minor fix

eejian97 committed:
remove unneeded import

eejian97 committed:
Merge pull request #96 from AY1920S1-CS2103T-F11-3/fix-analyse-password-2

Fix analyse password 2

jityong committed:
fixed bugs on sort and edit issue

jityong committed:
updated some notes commands

jityong committed:
printf("%p", &iArray[0]);
Merge branch 'update-ug' of https://github.com/AY1920S1-CS2103T-F11-3/main into update-ug

niqiukun committed:
Change the rename command to retain file extension

jityong committed:
added tip and warning logo to notes section of ug

jityong committed:
Merge pull request #97 from AY1920S1-CS2103T-F11-3/fix-notes-bugs

Fix notes bugs

niqiukun committed:
Change the user guide of rename file command

niqiukun committed:
Merge pull request #98 from AY1920S1-CS2103T-F11-3/rename-without-extension

Change the rename file command to retain file extension

niqiukun committed:
Merge pull request #66 from AY1920S1-CS2103T-F11-3/explain-encryption-util

Add explanation of encryption method in the developer guide

yhtingg committed:
Change default view to password

yhtingg committed:
Update Ui.png

niqiukun committed:
Update version number for prepare for release of v1.3

yhtingg committed:
Temporary comment out LogicManagerTest to pass build

yhtingg committed:
Merge pull request #100 from AY1920S1-CS2103T-F11-3/modify-default-view

Change default view to password

niqiukun committed:
Merge pull request #106 from AY1920S1-CS2103T-F11-3/update-release-1.3

Update version number to prepare for release of v1.3

niqiukun committed:
Merge pull request #60 from AY1920S1-CS2103T-F11-3/update-ug

Update UG

jityong committed:
updated notes ug

jityong committed:
Merge pull request #109 from AY1920S1-CS2103T-F11-3/update-notes-ug

updated notes ug

yhtingg committed:
Change card command formatting

yhtingg committed:
Merge pull request #115 from AY1920S1-CS2103T-F11-3/update-ug-card-commands

Change card command formatting

yhtingg committed:
Disallow adding of expired cards

yhtingg committed:
Fix duplicate prefix field bug for card

yhtingg committed:
Suppress unchecked warnings for fillReadParts

yhtingg committed:
Merge pull request #118 from AY1920S1-CS2103T-F11-3/fix-card-model-bugs

Fix card model bugs

hooncp committed:
Add security feature to clear clipboard only when app is closed through exit command

eejian97 committed:
added better exception handling for password generate

eejian97 committed:
fixed font error

niqiukun committed:
Change the command result message to new file after renaming

niqiukun committed:
Correct the usage message of move file command

niqiukun committed:
Change the sample file paths in file commands to be clearer

niqiukun committed:
Change the sample file paths in the user guide

niqiukun committed:
Fix parsing of move and rename commands

hooncp committed:
Refactor Description into PasswordDescription. Change equality from same description to same description and username

hooncp committed:
Fix documentation and make application error message more user friendly

niqiukun committed:
Refactor code in MainApp and add masking of master password

hooncp committed:
Remove clipboard from exit command because of test failure

hooncp committed:
Change documentation in application to make it more explicit

eejian97 committed:
added UI for analyse password

hooncp committed:
Add UI to enhance feature

hooncp committed:
Minor fix to phrasing

hooncp committed:
Merge pull request #170 from AY1920S1-CS2103T-F11-3/fix-password-branch2

Fix password branch2

hooncp committed:
Refine find password error throwing, with description parameters.

eejian97 committed:
added in keyboard analyser class

hooncp committed:
Fix goto command bug after find command

hooncp committed:
Change password value to be consistent with generate and analyse.

hooncp committed:
Add coverall to repo

hooncp committed:
Merge pull request #173 from AY1920S1-CS2103T-F11-3/add-coverall-branch

Add coverall to repo

hooncp committed:
Fix formatting of coverall badge

hooncp committed:
Merge pull request #174 from AY1920S1-CS2103T-F11-3/add-coverall-branch

Fix formatting of coverall badge

hooncp committed:
Fix duplicate fields in edit command and tag limit

hooncp committed:
Add new tests

niqiukun committed:
Merge pull request #169 from AY1920S1-CS2103T-F11-3/fix-file-commands

Fix file commands according to feedbacks from mock PE

niqiukun committed:
Merge pull request #171 from AY1920S1-CS2103T-F11-3/add-masterpwd-masking

Add masking of master password and apply double confirmation on create

niqiukun committed:
Add preview command to file manager

niqiukun committed:
Add some tests for model

hooncp committed:
Improve phrasing and format.

jityong committed:
improved implementation of undo-redo command and also informs the user which command is being undone/redone

hooncp committed:
Fix file find bug in goto command

hooncp committed:
Add tests for StringUtil and DateUtil

hooncp committed:
Merge pull request #176 from AY1920S1-CS2103T-F11-3/refine-password-branch

Refine password branch

hooncp committed:
Add test and support for passwordBookParser

yhtingg committed:
Change tag example for AddCardCommand

yhtingg committed:
Merge pull request #177 from AY1920S1-CS2103T-F11-3/modify-add-card-message

Change tag example for AddCardCommand

yhtingg committed:
Add support to delete cards by index

yhtingg committed:
Modify DeleteCardCommand message

yhtingg committed:
Modify Card Description regex and message

yhtingg committed:
Add support for ReadCardCommand to read by index

yhtingg committed:
Fix lint errors

yhtingg committed:
Modify card description error message

yhtingg committed:
Update ReadCardCommand equals method

niqiukun committed:
Refactor code in accordance with SLAP principle

niqiukun committed:
Fix formatting in build.gradle

eejian97 committed:
minor refactoring, moved dictionary test to new module

eejian97 committed:
fixed StrengthAnalyser bug, minor refactoring

jityong committed:
wip: updated dg for notes

jityong committed:
wip: updated dg for notes

yhtingg committed:
Merge pull request #178 from AY1920S1-CS2103T-F11-3/include-card-read-by-index

Include card read by index

niqiukun committed:
Add test for encrypt file command

niqiukun committed:
Merge pull request #180 from AY1920S1-CS2103T-F11-3/add-preview-command

Add preview file command and improve code quality

eejian97 committed:
fixed bug for strength analyser, fixed bug for generateutil, added in index for tableview

eejian97 committed:
Merge branch 'master' into fix-analyse-password-3

eejian97 committed:
resolved conflict

eejian97 committed:
added EoF for darktheme.css

eejian97 committed:
Merge pull request #181 from AY1920S1-CS2103T-F11-3/fix-analyse-password-3

Fix analyse password 3

jityong committed:
handle duplicated prefixes in the commands

jityong committed:
commented out some testcases for parsing temp

jityong committed:
fixed test cases for handling duplicate prefixes in a command

eejian97 committed:
minor refactoring, better exception handling

jityong committed:
updated notes sorting to allow multiple sorting fields

eejian97 committed:
wip : dg for analyse and generate

jityong committed:
updated notes ug on adding and sorting commands

jityong committed:
changed error thrown by duplicated prefixes and updated it test cases

jityong committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F11-3/main into update-note-undo-redo

jityong committed:
fixed travis test cases

eejian97 committed:
added PPP

jityong committed:
Merge branch 'update-note-undo-redo' of https://github.com/AY1920S1-CS2103T-F11-3/main into update-notes-sort

jityong committed:
merged with update-note-undo-redo branch

yhtingg committed:
Change card sample data to temporarily fix bug

yhtingg committed:
Merge pull request #185 from AY1920S1-CS2103T-F11-3/fix-card-sample-data

Change card sample data to temporarily fix bug

jityong committed:
fixed output from sort command

eejian97 committed:
fixed minor bug

eejian97 committed:
Merge pull request #188 from AY1920S1-CS2103T-F11-3/fix-main-window

fixed minor bug

jityong committed:
handled duplicate conditions in sort

jityong committed:
updated travis fix

jityong committed:
made open note a undoable command as it changes an attribute of the note

eejian97 committed:
added better exception handling for generate

eejian97 committed:
made parseBool case insensitive

eejian97 committed:
added EoF

eejian97 committed:
Merge pull request #191 from AY1920S1-CS2103T-F11-3/add-eejian-ppp

refactored code

eejian97 committed:
added in detailed explanation for analysis
added assertion

jityong committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F11-3/main into update-notes-sort

jityong committed:
Merge pull request #192 from AY1920S1-CS2103T-F11-3/update-notes-sort

Updated notes and fixed some general bugs

jityong committed:
fixed duplicate title name for notes. does not allow same titles for different names [case-insensitive]

eejian97 committed:
added ResultOutcome class

eejian97 committed:
integrated result outcome, added better SLAP for code

yhtingg committed:
Fix bug to allow expired cards to reside in CardBook

yhtingg committed:
Improve exception catching for ReadCardCommand

yhtingg committed:
Improve formatting of ReadCardCommand message usage

yhtingg committed:
Edit Tag message constraints

eejian97 committed:
refactored code for more SLAP

eejian97 committed:
finished JavaDoc

yhtingg committed:
Add support for copying card numbers to clipboard

jityong committed:
updated notes DG with undo/redo (wip)

jityong committed:
refactored undo/redo code in notes

jityong committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F11-3/main into update-notes-sort

jityong committed:
Merge branch 'master' into update-notes-sort

jityong committed:
Merge pull request #193 from AY1920S1-CS2103T-F11-3/update-notes-sort

Refactored code and fixed duplicate note title bug

jityong committed:
updated notes undo/redo feature for DG

yhtingg committed:
Add UI warnings for expiring and expired cards

yhtingg committed:
Fix lint errors and add respective Javadocs

yhtingg committed:
Limit number of tags to 5 for Cards

yhtingg committed:
Remove redundant import

yhtingg committed:
Merge pull request #194 from AY1920S1-CS2103T-F11-3/enhance-card-features

Enhance card features

yhtingg committed:
Update card commands in UG

jityong committed:
wip: adding sort notes DG

jityong committed:
merged with master

eejian97 committed:
added in test for basematch

jityong committed:
updated sort notes DG wip

eejian97 committed:
added in test cases

eejian97 committed:
Merge branch 'update-UG-DG' of https://github.com/AY1920S1-CS2103T-F11-3/main into update-UG-DG

jityong committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F11-3/main into update-notes-sort

jityong committed:
updated picture on aboutus

jityong committed:
Merge pull request #195 from AY1920S1-CS2103T-F11-3/update-notes-sort

updated photo for aboutus

jityong committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F11-3/main into add-notes-test

jityong committed:
refactored note model to remove sortByCond class

hooncp committed:
Change name to match requirements and minor rephrasing

hooncp committed:
Refine passwordbook for demo

hooncp committed:
Fix bug where NIL is copied in copy website command

hooncp committed:
Merge pull request #196 from AY1920S1-CS2103T-F11-3/refine-for-demo

Refine for demo

hooncp committed:
Add pictures to UG of password

hooncp committed:
Add clearer instructions to UG

niqiukun committed:
Upload project portfolio page

niqiukun committed:
Make preview text area extendable

jityong committed:
added test cases for notes commands and model

eejian97 committed:
Merge branch 'update-UG-DG' of https://github.com/AY1920S1-CS2103T-F11-3/main into update-UG-DG

eejian97 committed:
wip : eejian-UGDG,
updated UG with pictures
added UG logo
updated UG intro

jityong committed:
Merge branch 'update-UG-DG' of https://github.com/AY1920S1-CS2103T-F11-3/main into update-UG-DG

jityong committed:
update notes DG

jityong committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F11-3/main into update-UG-DG

eejian97 committed:
fixed copypasswordcommandsequencediagram

niqiukun committed:
Add reference links to the user guide

jityong committed:
updated changes and images to notes ug

niqiukun committed:
Do minor adjustments

eejian97 committed:
updated PPP

hooncp committed:
Add password feature to UG/DG

niqiukun committed:
Merge branch 'update-UGDG-niqiukun' into update-UG-DG

niqiukun committed:
Add use of "we" in developer guide

hooncp committed:
Merge branch 'PPP-CP' into update-UG-DG

eejian97 committed:
added ppp

eejian97 committed:
Merge branch 'update-UG-DG' of https://github.com/AY1920S1-CS2103T-F11-3/main into update-UG-DG

# Conflicts:
#	docs/DeveloperGuide.adoc

eejian97 committed:
added copypasswordsequence diagram

jityong committed:
updated notes UG and DG and created PPP pdf

jityong committed:
updated notes UG and DG and created PPP pdf

eejian97 committed:
added tests

eejian97 committed:
Merge pull request #198 from AY1920S1-CS2103T-F11-3/fix-analyse-password-4

Fix analyse password 4

yhtingg committed:
Edit UG and partial DG for PPP

jityong committed:
updated notes UG and DG and changed commitNote() to commitNoteBook()

jityong committed:
Merge branch 'update-UG-DG' of https://github.com/AY1920S1-CS2103T-F11-3/main into update-UG-DG

jityong committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F11-3/main into add-notes-test

hooncp committed:
Final changes to PPP

hooncp committed:
Minor update

hooncp committed:
Last minor update

yhtingg committed:
Add UML diagram and Design Considerations for cards

jityong committed:
pulled from update-ug-dg branch

jityong committed:
Merge branch 'update-UG-DG' of https://github.com/AY1920S1-CS2103T-F11-3/main into add-notes-test

hooncp committed:
Minor changes to code link

eejian97 committed:
added ppp

eejian97 committed:
Merge branch 'update-UG-DG' of https://github.com/AY1920S1-CS2103T-F11-3/main into update-UG-DG

hooncp committed:
Merge branch 'master' into improve-test-coverage-password-branch

hooncp committed:
Add test for parserUtil

eejian97 committed:
accounted for no passwords to analyse

eejian97 committed:
Merge pull request #199 from AY1920S1-CS2103T-F11-3/fix-analyse-password-4

accounted for no passwords to analyse

jityong committed:
updated notes test

jityong committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F11-3/main into add-notes-test

eejian97 committed:
edited PPP intro

eejian97 committed:
Merge branch 'master' into update-UG-DG

yhtingg committed:
Add in app screenshots and modify class diagram

eejian97 committed:
resolve travis build problem

eejian97 committed:
Merge branch 'update-UG-DG' of https://github.com/AY1920S1-CS2103T-F11-3/main into update-UG-DG

eejian97 committed:
Merge pull request #201 from AY1920S1-CS2103T-F11-3/update-UG-DG

Update ug dg

hooncp committed:
Add more integration test for AddPasswordCommand

hooncp committed:
Merge pull request #175 from AY1920S1-CS2103T-F11-3/improve-test-coverage-password-branch

Add new tests

hooncp committed:
Add new line to one command

jityong committed:
final changes to PPP

jityong committed:
Merge branch 'update-UG-DG' of https://github.com/AY1920S1-CS2103T-F11-3/main into update-UG-DG

jityong committed:
final changes to UG notes

hooncp committed:
Minor fix to PPP

hooncp committed:
Remove coverall

jityong committed:
final changes to UG

eejian97 committed:
rename PPP

eejian97 committed:
renamed PPP

hooncp committed:
Merge pull request #202 from AY1920S1-CS2103T-F11-3/minor-fix

Add new line to one command

jityong committed:
resolve conflict

jityong committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F11-3/main into add-notes-test

jityong committed:
resolved conflicts

yhtingg committed:
Complete DG for cards

yhtingg committed:
Complete PPP

eejian97 committed:
fixed travis issue

jityong committed:
fix travis

eejian97 committed:
Merge branch 'update-UG-DG' of https://github.com/AY1920S1-CS2103T-F11-3/main into update-UG-DG

eejian97 committed:
solve travis build

eejian97 committed:
Merge pull request #203 from AY1920S1-CS2103T-F11-3/update-UG-DG

Update ug dg

jityong committed:
manually undo some conflicts

yhtingg committed:
Final tweaks to PPP

jityong committed:
Merge branch 'master' into add-notes-test

jityong committed:
Merge pull request #200 from AY1920S1-CS2103T-F11-3/add-notes-test

Add notes test

yhtingg committed:
Add tests for cards

hooncp committed:
Fix DG minor

niqiukun committed:
Do final edit on UG

yhtingg committed:
Add tests for cards

hooncp committed:
Merge pull request #205 from AY1920S1-CS2103T-F11-3/hothothot-fix

Fix DG minor

niqiukun committed:
Merge pull request #206 from AY1920S1-CS2103T-F11-3/final-edit-on-ug

Do final edit on the user guide

yhtingg committed:
Merge pull request #207 from AY1920S1-CS2103T-F11-3/card-tests

Card tests

yhtingg committed:
Merge pull request #204 from AY1920S1-CS2103T-F11-3/update-UG-DG

Final tweaks to PPP

niqiukun committed:
Change version number to v1.4

niqiukun committed:
Merge pull request #208 from AY1920S1-CS2103T-F11-3/change-version-1.4

Change version number to v1.4

jityong committed:
Rename leowjityong.adoc to jityong.adoc

hooncp committed:
Rename img file

hooncp committed:
Merge pull request #209 from AY1920S1-CS2103T-F11-3/rename-img-file

Rename img file
